app/src/main/java/io/intrepid/contest/screens/contestjudging/scoreentrydetail/expandablerecycler/ScorableCategoryViewHolder.java

```diff
@@ -0,0 +1,116 @@
+package io.intrepid.contest.screens.contestjudging.scoreentrydetail.expandablerecycler;
+
+import android.os.Handler;
+import android.support.annotation.StringRes;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+import android.widget.RatingBar;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import io.intrepid.contest.R;
+import io.intrepid.contest.models.Category;
+
+import static android.view.View.GONE;
+import static android.view.View.VISIBLE;
+
+public class ScorableCategoryViewHolder extends RecyclerView.ViewHolder implements ExpandableViewHolder, RatingBar.OnRatingBarChangeListener {
+    private static final long COLLAPSE_DELAY = 800;
+    private final ExpandClickListener listener;
+    @BindView(R.id.category_item_title)
+    TextView categoryTitleField;
+    @BindView(R.id.expandable_relative_layout)
+    View expandableLinearLayout;
+    @BindView(R.id.editable_rating_bar_category)
+    RatingBar ratingBar;
+    @BindView(R.id.category_score_status_textview)
+    TextView scoreStatusTextView;
+
+    private Category category;
+    private boolean reveal;
+    @BindView(R.id.category_item_description)
```

Comment:
Very nit picky, but you can group the butterknife variables together so the other variables are more visible